python/pyproject.toml

```
@@ -18,8 +18,8 @@
 [build-system]
 requires = [
     "cython >= 0.29",
-    "numpy==1.14.5; python_version<'3.7'",
-    "numpy==1.16.0; python_version>='3.7'",
+    "numpy==1.16.6; python_version<='3.8'",
```

```suggestion
    "numpy==1.16.6; python_version<'3.9'",
```
I don't know how exactly those versions are matched (eg is 3.8.5 <= 3.8 ?), so this might be clearer